planemo/conda.py

```diff
@@ -153,7 +153,7 @@ def collect_conda_target_lists_and_tool_paths(ctx, paths, recursive=False, found
 
 def tool_source_conda_targets(tool_source):
     """Load CondaTarget object from supplied abstract tool source."""
-    requirements, _ = tool_source.parse_requirements_and_containers()
+    requirements, *_ = tool_source.parse_requirements_and_containers()
```

Comment:
What is this syntax doing? Never have seen this before.

Reply:
https://stackoverflow.com/questions/13726155/is-an-acceptable-way-to-ignore-arguments-in-python